### PR TITLE
Subscriber offer tweaks

### PR DESCRIPTION
**Pull request by mattandrews**
## Toggle subscriber number

memb

#### Before
screen shot 2015-05-07 at 12 43 37
#### After
screen shot 2015-05-07 at 12 42 37

#### Before
screen shot 2015-05-07 at 12 43 43
#### After
screen shot 2015-05-07 at 12 46 28

#### Before
screen shot 2015-05-07 at 12 44 05
#### After
screen shot 2015-05-07 at 12 44 14

### CODE REVIEW

**Comment by mattandrews** [comment]
cc @davidrapson

File: frontend/app/views/fragments/form/subscriberNumber.scala.html [hunk 1]
```diff
@@ -3,12 +3,23 @@
 <fieldset class="fieldset">
 
     <div class="fieldset__heading">
-        <h2 class="fieldset__headline">Subscribers pay nothing for 6 months</h2>
+        <h2 class="fieldset__headline">Subscribers pay nothing<br /> for six months</h2>
```
Comment (mattandrews) [inline frontend/app/views/fragments/form/subscriberNumber.scala.html]: This was partly for aesthetic reasons and partly to fit Guardian style (six not 6).

File: frontend/app/model/Faq.scala [hunk 2]
```diff
@@ -186,6 +186,32 @@ object Faq {
...
+    Item("When do I need to take up this offer?",
+      Html("This offer is open until further notice. However, we reserve the right at any time to modify or discontinue, temporarily or permanently, this offer without prior notice."),
+      "when-start-offer"
+    ),
+    Item("Does this affect my subscription to the Guardian?",
+      Html(s"No, this offer does not affect your subscription but if you have any concerns then do <a href='mailto:${Email.membershipSupport}'>get in touch with our membership team</a>."),
+      "affect-subscription"
+    ),
+    Item("I have a query about my subscription. Who do I contact?",
+      Html("""If there is anything you’d like to know about your subscription, please contact us on the numbers below. Please ensure that you quote your Subscriber ID when contacting customer services.
```
Comment (davidrapson) [inline frontend/app/model/Faq.scala]: Hmm, never really thought about what would happen if we needed larger blocks of HTML in here, seems OK.

File: frontend/app/views/fragments/form/subscriberNumber.scala.html [hunk 3]
```diff
@@ -3,12 +3,23 @@
...
+                    data-toggle-label="Hide"
+                    data-toggle="js-sub-offer-help"
+                    data-metric-trigger="click"
+                    data-metric-category="registration"
+                    data-metric-action="toggle-subscriber-help"
+                    data-toggle-icon="triangle-up">
+                @fragments.inlineIcon("triangle-down", Seq("u-align-right"))
+                Need help redeeming this offer?
+            </button>
+            <div id="js-sub-offer-help" data-toggle-hidden>
```
Comment (davidrapson) [inline frontend/app/views/fragments/form/subscriberNumber.scala.html]: I'd probably remove the 'js-' prefix here as it's an ID and presumably it's only ever referenced in the `data-toggle` above.

**Comment by davidrapson** [comment]
:+1: one minor comment, but looks good to me.

File: frontend/app/views/fragments/form/subscriberNumber.scala.html (hunk 3 above)
Comment (mattandrews) [inline frontend/app/views/fragments/form/subscriberNumber.scala.html]: Good call, done.

**Comment by mattandrews** [comment]
Cool – okay with you, @TesterSpike?

**Comment by mattandrews** [comment]
Moved the final question.

**Comment by mattandrews** [comment]
Added T&Cs – @davidrapson could you sign off the last few commits here?

**Comment by davidrapson** [comment]
:+1: LGTM

**Pull request merged**